lib/datasource/remote/model/region_model.dart

```diff
@@ -50,4 +52,20 @@ class RegionModel {
       createdAt: json['created_at'],
     );
   }
+
+  RegionModel addImageUrlToModel(String imageUrl) {
```

Comment:
Since region image URL comes from fire. we need to add it to the Chain view model later on. Probably at the use case level